.github/workflows/deploy-website.yaml

```diff
@@ -35,7 +35,7 @@ jobs:
         with:
           path: ./website/_site
   deploy:
-    if: github.ref == 'refs/heads/master'
+    if: github.ref == 'refs/heads/master' || github.ref == 'refs/heads/website'
```

Comment:
Also requires a change in our environment settings to allow this branch to deploy.